frontend/src/pages/Ambassadors.jsx

```diff
@@ -3,10 +3,12 @@ import "./styles/Ambassadors.scss";
 import FilterTab from "../components/FilterTab";
 import ProfileCard from "../components/ProfileCard";
 import axios from "axios";
+import { useSelector } from "react-redux";
 const { apiCall } = require("../conf");
 
 function Ambassadors() {
   const [ambassadors, setAmbassadors] = useState([]);
+  const filter = useSelector(state => state.filter);
```

Comment:
(It's a **very** bad practice to name a variable with the name of a JS function)

Reply:
OK i will fix this the next PR